src/sentry/api/endpoints/organization_users.py

```diff
@@ -32,21 +34,29 @@ def get(self, request, organization):
         :auth: required
         """
         projects = self.get_projects(request, organization)
-        qs = (
-            OrganizationMember.objects.filter(
-                user__is_active=True,
-                organization=organization,
-                teams__projectteam__project__in=projects,
+
+        with sentry_sdk.start_span(op="OrganizationUsersEndpoint.get_members") as span:
+            qs = (
+                OrganizationMember.objects.filter(
+                    user__is_active=True,
+                    organization=organization,
+                    teams__projectteam__project__in=projects,
+                )
+                .select_related("user")
+                .prefetch_related(
+                    "teams", "teams__projectteam_set", "teams__projectteam_set__project"
+                )
+                .order_by("user__email")
+                .distinct()
             )
-            .select_related("user")
-            .prefetch_related("teams", "teams__projectteam_set", "teams__projectteam_set__project")
-            .order_by("user__email")
-            .distinct()
-        )
+            organization_members = list(qs)
```

Comment:
interesting.  you think that would change things?

Reply:
Nah, it's just to pull it inside the span. In terms of execution, it should be a null change: `list(qs)` was being called before passing it to `serialize`.

Reply:
ah, make sense.